_clients/logstash/read-from-opensearch.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Read from OpenSearch
+parent: Logstash
+nav_order: 220
+---
+
+# Read from OpenSearch
+
+As we ship Logstash events to an OpenSearch cluster using the OpenSearch output plugin. We can also perform read operations on an OpenSearch cluster and load data into Logstash using the OpenSearch input plugin.
+
+The OpenSearch input plugin helps to read the search query results performed on an OpenSearch cluster and load them into Logstash. This is useful for replaying test logs, reindexing, etc. Users can schedule this ingestion to run periodically using 
+cron syntax(using `schedule` configuration setting) or by just running the query once to load data into Logstash.
+
+
+
+## OpenSearch input plugin
+
+To run the OpenSearch input plugin, add the configuration in your `pipeline.conf` file like the sample configuration shown below:
```

Comment:
"To run the OpenSearch input plugin, add the configuration to the `pipeline.conf` file within your Logstash's `config` folder. The example example below runs the `match_all` query filter and loads in data once."

And then we can delete the sentence that follows the yml snippet.

Reply:
ack